lib/features/email/presentation/email_controller.dart

```diff
@@ -614,6 +614,7 @@ class EmailController extends BaseController {
 
   void backToThreadView(BuildContext context) {
     mailboxDashBoardController.clearSelectedEmail();
+    mailboxDashBoardController.disableSearch();
```

Comment:
in the aspect of UX, IMO no problem when user want to search when he stays in `EmailView`